content/guide/install-uninstall.md

```diff
@@ -10,7 +10,7 @@ type: essay
 
 ## macOS Installation
 
-Open your Terminal {{< q-glossary "command-line shell" >}} (found in your Applications/Utilities folder) and follow the steps below. First, install support software for Quire, and then Quire itself. If you are new to the command-line, read our tutorial on [“Working in a Command-line Shell”](/tutorial#1-working-in-a-command-line-shell).
+Open your Terminal {{< q-glossary "command-line shell" >}} (found in your Applications/Utilities folder or by pressing Command–Space Bar and type “Terminal” ) and follow the steps below. First, install support software for Quire, and then Quire itself. If you are new to the command-line, read our tutorial on [“Working in a Command-line Shell”](/tutorial#1-working-in-a-command-line-shell).
```

Comment:
Can you search the other parts of the documentation to see if we have a standard way of writing out keyboard shortcuts? You use a hyphen in "Command–Space Bar" but I typically use a plus sign, like "Command+Space Bar". Happy to go either way, I just want to make sure it's consistent.

Reply:
Absolutely! I did cut-and-paste that phrasing from elsewhere in the documentation but happy to do a deeper search for consistency. This dovetails nicely with a question I had. I read somewhere that it is ideal to have your documentation appear all on one page (where you can jump to certain parts) that way the entire document is searchable. Right now, we need to click through to the individual sections and search within each of them separately. I can definitely see someone trying to search for a keyword. Is there a way to make the whole doc searchable? It's a bigger question that's on a longer list I'm keeping track of. Just feels relevant to mention here.

Reply:
Ok, I looked and didn't see any other instance of "Command-Space" or "Command+Space". I did, however, see "Control-C" to stop the preview from running. So it looks like there is a little bit of consistency with the "-".  Also, when Apple refers to these commands they use the "-" https://support.apple.com/en-us/HT201236. So my vote is to stick with that. Thank you!

Reply:
The live site (https://gettypubs.github.io/quire/) has search functionality (in the upper right corner) and the way it works now, that search is actually more effective on multiple smaller pages than it would be on a single page. I'd also be weary about trying to maintain an enormous single Markdown file with all the documentation. But I could be convinced otherwise. Do you have examples?

Reply:
Ok. I'm good with the hyphen. Thanks